packages/frontend-2/pages/server-management/pending-invitations.vue

```diff
@@ -0,0 +1,272 @@
+    <Portal to="navigation">
+      <HeaderNavLink
+        :to="'/server-management'"
+        name="Server Management"
+      ></HeaderNavLink>
+      <HeaderNavLink
+        :to="'/server-management/pending-invitations/'"
+        name="Pending Invitations"
+      ></HeaderNavLink>
+    </Portal>
+
+    <div
+      class="flex flex-col md:flex-row space-y-2 space-x-2 justify-between mb-4 md:items-center h-8"
+    >
+      <div>
+        <h5 class="h4 font-bold">Pending Invitations</h5>
+      </div>
+    </div>
+
+    <div class="flex items-center gap-8 h-10">
+      <FormTextInput
+        v-model="searchString"
+        size="lg"
+        name="search"
+        :custom-icon="MagnifyingGlassIcon"
+        color="foundation"
+        full-width
+        search
+        :show-clear="!!searchString"
+        placeholder="Search Users"
+        class="rounded-md border border-outline-3"
+      />
+      <div class="flex items-center gap-2 text-foreground text-sm shrink-0">
+        <span class="shrink-0">1-50 of 350</span>
+        <div class="flex gap-1">
+          <ChevronLeftIcon class="h-8 w-8" />
+          <ChevronRightIcon class="h-8 w-8" />
+        </div>
+      </div>
+    </div>
+
+    <Table
+      :headers="[
+        { id: 'email', title: 'Email' },
+        { id: 'invitedBy', title: 'Invited By' },
+        { id: 'resend', title: '' }
+      ]"
+      :items="users"
+      :buttons="[
+        { icon: TrashIcon, label: 'Delete', action: openDeleteInvitationDialog }
+      ]"
+      :column-classes="{
+        email: 'col-span-5',
+        invitedBy: 'col-span-4',
+        resend: 'col-span-3'
+      }"
+    >
+      <template #email="{ item }">
+        {{ item.email }}
+      </template>
+
+      <template #invitedBy="{ item }">
+        <div class="flex items-center gap-2">
+          <img
+            :src="item.profilePicture"
+            :alt="'Profile picture of ' + item.invitedBy"
+            class="w-6 h-6 rounded-full"
+          />
+          {{ item.invitedBy }}
+        </div>
+      </template>
+
+      <template #resend="{ item }">
+        <button class="font-semibold text-primary" @click="resendInvitation(item)">
+          Resend Invitation
+        </button>
+      </template>
+    </Table>
+
+    <DeleteInvitationDialog
+      v-model:open="showDeleteInvitationDialog"
+      :user="user ?? userToModify"
+      title="Delete Invitation"
+      :buttons="[
+        {
+          text: 'Delete',
+          props: { color: 'danger', fullWidth: true },
+          onClick: deleteConfirmed
+        },
+        {
+          text: 'Cancel',
+          props: { color: 'secondary', fullWidth: true, outline: true },
+          onClick: closeDeleteInvitationDialog
+        }
+      ]"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import {
+  MagnifyingGlassIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon
+} from '@heroicons/vue/20/solid'
+
+import Table from '../../components/server-management/Table.vue'
```

Comment:
1. Relative imports
2. Manually imported vue components (Nuxt has auto-imports for local components)